docs/quickstart/ccd.md

```diff
@@ -0,0 +1,246 @@
+    decrypter-a268aea6-4b5b-4241-b833-9a84f4f44bc4   1/1     1            1           93m
+
+You could use `kubectl describe deployment decrypter...` to see the annotations on the deployment, and see that it is
+indeed processing your `test-2` stream
+
+### Sending events with the cli
+
+In order to use the cli to simulated data, you'll need to send the events via telepresence, using the `events-api-url`
+parameter (that you could also store in the strm config file (`strm context config`).
+
+    strm simulate random-events test --events-api-url=http://event-gateway.strmprivacy/event --interval 5
+    Sent 874 events
+    Sent 1809 events
+    Sent 2726 events
+    ...
+
+### Listening on the web-socket via the cli
+
+    strm listen web-socket test --web-socket-url ws://web-socket.strmprivacy/ws
+
+    {"strmMeta": {"eventContractRef": "strmprivacy/example/1.3.0", "nonce": -447993628, "timestamp": 1652181230883, "keyLink": "7573fc76-ae34-4c49-a3fd-d552b677ffa1", "billingId": "strmprodccdtest1908747604", "consentLevels": [0, 1, 2, 3]}, "uniqueIdentifier": "AQAsswoVM2q6Q6+eeTb5Qe61xBHTaAZZMVCh+vDf", "consistentValue": "AQAsswoPYW8+VGwOZvfh+FmSEh2UoVTRNkNWlyQOpwA=", "someSensitiveValue": "AQAsswpQCKDPUYNls3hy13IllL5vd4bz/X3rEsBI0TEV", "notSensitiveValue": "not-sensitive-64"}
+    {"strmMeta": {"eventContractRef": "strmprivacy/example/1.3.0", "nonce": -45323834, "timestamp": 1652181230893, "keyLink": "9ced67fc-9227-4fb2-846b-1eed521db941", "billingId": "strmprodccdtest1908747604", "consentLevels": [0, 1, 2]}, "uniqueIdentifier": "AWzkJjeWufjZehXNpWLUQ63CG1O8jxCU6MOd3Seo", "consistentValue": "AWzkJjd3cB+36sCrtT6va3YLjCa5qw55Iy5/HevIcVU=", "someSensitiveValue": "AWzkJjcsrjNHbZCAjlEYCBdlpOVK8+eXIx6BSBDxH8VQ", "notSensitiveValue": "not-sensitive-42"}
+
+Or a derived stream
+
+    strm listen web-socket test-2 --web-socket-url ws://web-socket.strmprivacy/ws
+
+    {"strmMeta": {"eventContractRef": "strmprivacy/example/1.3.0", "nonce": -1742873135, "timestamp": 1652181297380, "keyLink": "556c1be7-4332-4058-9d36-5e3e5a66e121", "billingId": "strmprodccdtest1908747604", "consentLevels": [0, 1, 2]}, "uniqueIdentifier": "unique-81", "consistentValue": "session-922", "someSensitiveValue": "AQM0jlnxbeNZSJzvJWLpnMjyYET1Jb1Yz+5yZVB5i6Dq", "notSensitiveValue": "not-sensitive-78"}
+
+### Kafka
+Install [Confluent client tools][confluent]. You don't have to start Confluent, you only need to add the `bin` directory
+of the unpacked confluent tar file  to your `$PATH`.
+
+
+> There is currently no supported way for non Strmprivacy engineers to determine the Kafka topic name.
+  Exposing this information is a priority.
+
+You can access the input topic of a decrypter as follows
+
+    kubectl get deployments.apps decrypter-... -o=jsonpath='{.metadata.annotations.input-topic}'
+
+    stream-a268aea6-4b5b-4241-b833-9a84f4f44bc4
+
+I haven't thought of a way to access the topic of a derived stream yet. Ask the Strm support engineers, they can access
```

Comment:
There should not be "I" in such a text as we speak on behalf of the company. Also, there is no visible author, so who is "I"?